### DISCUSSION TITLE
Release cadence?

### DISCUSSION THREAD

**Discussion by bryankaylor** [ansys/pyaedt#3496] [General]
Hi- I was curious if there's any planned release cadence for new versions of PyAEDT?

I'm testing a tool that installs PyAEDT via pip. Occasionally there will be an issue in the installed package, and it might be helpful to know when a version with the fix will be available in pip, so I can decide whether it's better to manually patch or wait and instruct the users to upgrade their package.

Thank you!

**Comment by MaxJPRey** [comment]
Hi @bryankaylor,
Thanks for your interest in PyAEDT.

We don't have any planned release.
We mostly try to do a release every week or so, as much as we can.
That will include all the last improvements, fix and new features.

If you want to get notified everytime a new release is available here are some steps to reproduce:
- Go to https://github.com/ansys/pyaedt

- Click on the `Watch` button.
<img width="214" alt="image" src="https://github.com/ansys/pyaedt/assets/87315832/084f72fa-faca-4698-a000-ca8300b08194">

- Toggle the `Custom` option `Releases`
<img width="186" alt="image" src="https://github.com/ansys/pyaedt/assets/87315832/e9de6e7a-5b4f-4a3f-b627-20a1b3df0ec8">
<img width="185" alt="image" src="https://github.com/ansys/pyaedt/assets/87315832/20529a17-f553-4a38-9b8b-f8215793239d">


Then you will get automatic notifications and you will be able to click on them to see all the PRs merged in the last release.


I hope it helps.
Thanks

**Reply by bryankaylor** [reply]
Sounds good, thank you for the info!
